client.go

```diff
@@ -229,8 +229,18 @@ func (client *Client) SetConfigFile(fpath string) error {
 	return nil
 }
 
+// SetTessdataPrefix sets path to the models directory.
+// Environment variable TESSDATA_PREFIX is used as default.
+func (client *Client) SetTessdataPrefix(prefix string) error {
+	if prefix == "" {
+		return fmt.Errorf("tessdata prefix could not be empty")
```

Comment:
Do you really think there is NO usecase of setting an empty string?
I'm not 100 % sure.

Reply:
If we will treat empty string as a valid parameter, tesseract will try to lookup models in nowhere and will exit with recognition error since no language models has been found.

Reply:
OK, then my opinion is that kicking out the empty value and letting tesseract fail are different experience for applications who use `gosseract`.

Do you think those 2 are the same?

Reply:
Previously `gosseract` users has no ability to change `TESSDATA_PREFIX` from go code at all (omit environment changing way), so that decision is totally up to you. As for me, `gosseract` is built for utilitarian usage of tesseract, not for it's development, which i believe the only case when you **need** to made tesseract to fail recognition by not providing language models.

Reply:
I do not understand. There are two ways which will lead to fail. Prefix has `*string` type so we can distinct empty and never assigned values. There is no difference, to set prefix as empty string explicitly or leave it `nilptr` value. If `TESSDATA_PREFIX` env is empty, both ways will lead to failure. 
Since empty string is a default value for Go string there is no reason to give to user another way to do same thing. Since it doesn't solve any problem, it's useless.